enterprise_subsidy/apps/subsidy/models.py

```diff
@@ -296,64 +323,99 @@ class AccessMethods:
     REQUEST = 'request'
 
 
-class SubsidyAccessPolicy(TimeStampedModelWithUuid):
+class SubsidyAccessPolicy(TimeStampedModel):
```

Comment:
this code was part of the example code but it is meant for the subsidy access policy API, we can remove this.